spec.html

```diff
@@ -20665,6 +21279,230 @@ <h1>Static Semantics: PrototypePropertyNameList</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-class-definitions-static-semantics-allprivateidentifiersvalid" type="sdo" aoid="AllPrivateIdentifiersValid">
+      <h1>Static Semantics: AllPrivateIdentifiersValid</h1>
+      <p>With parameter _names_.</p>
+      <emu-grammar>MemberExpression : MemberExpression `.` PrivateIdentifier</emu-grammar>
+      <emu-alg>
+        1. If StringValue of |PrivateIdentifier| is in _names_, return *true*.
```

Comment:
```suggestion
        1. If _names_ contains the StringValue of |PrivateIdentifier|, return *true*.
```

(Same for the other occurrences of this pattern below, of course.)